packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -124,7 +124,7 @@ def registry_cluster(cluster, request):
             'healthChecks': [{
                 'protocol': 'COMMAND',
                 'command': {'value': 'curl -fsSlv https://{}/v2/test_server/manifests/latest'.format(cluster.registry)},
-                'gracePeriodSeconds': 20,
+                'gracePeriodSeconds': 300,
                 'intervalSeconds': 20,
                 'timeoutSeconds': 10,
                 'maxConsecutiveFailures': 3}],
```

Comment:
The above 5 lines are all set to the Marathon healthCheck defaults... while it is nice to be explicit, it's also nice to keep the code small and readable... consider removing.